policyengine_us/parameters/gov/states/mt/tax/income/exemptions/interest_exemption.yaml

```diff
@@ -0,0 +1,16 @@
+description: Georgia counts the following income sources for the retirement exclusion.
+metadata:
+  label: Georgia retirement income sources
+  period: year
+  unit: list
+  reference: 
+  - title: 2021 IT-511 Individual Income Tax Booklet
+    href: https://dor.georgia.gov/document/booklet/2021-it-511-individual-income-tax-booklet/download#page=18
```

Comment:
```suggestion
    href: https://dor.georgia.gov/document/booklet/2021-it-511-individual-income-tax-booklet/download # Page=18
```

Reply:
i am also not sure if the pages are correct

Reply:
This is Montana's interest exemption file. I don't know why it is here. I will delete it.

Reply:
This is a Montana interest exemption file. I don't know why it is here. I will delete it